pkg/interface/src/logic/lib/omnibox.js

```diff
@@ -99,9 +99,11 @@ export default function index(contacts, associations, apps, currentGroup, groups
   Object.keys(associations).filter((e) => {
     // skip apps with no metadata
     return Object.keys(associations[e]).length > 0;
-    }).map((e) => {
-      // iterate through each app's metadata object
-      Object.keys(associations[e]).map((association) => {
+  }).map((e) => {
+    // iterate through each app's metadata object
+    Object.keys(associations[e])
+      .filter((association) => !associations[e][association].metadata.hidden)
```

Comment:
This should probably be a bit safer I think, it's quite nested

Reply:
Accessing using the same pattern found in the following map? Not sure what else to do.